docs/heasarc/heasarc.rst

```diff
@@ -11,292 +11,272 @@ This is a python interface for querying the
+    cmbmaster  LAMBDA Cosmic Microwave Background Experiments Master Catalog
+    erosmaster eROSITA Observations Master Catalog
 
-Note that the ``SEARCH_OFFSET_`` column will always be included in the results.
-If a column name is passed to the ``fields`` parameter which does not exist in
-the requested mission table, the query will fail. To obtain a list of available
-columns for a given mission table, do the following:
+If you do not know the name of the table you are looking for, you can use the ``keywords`` 
+parameter in `~astroquery.heasarc.HeasarcClass.tables`. For example, if you want to find all tables that 
+are related to Chandra, you can do:
 
 .. doctest-remote-data::
 
-    >>> cols = heasarc.query_mission_cols(mission='rosmaster')
-    >>> print(cols)
-    ['SEQ_ID', 'INSTRUMENT', 'EXPOSURE', 'NAME', 'RA', 'DEC', 'START_TIME', 'END_TIME', 'AO', 'BII', 'CLASS', 'FILTER', 'FITS_TYPE', 'INDEX_ID', 'LII', 'PI_FNAME', 'PI_LNAME', 'PROC_REV', 'PROPOSAL_NUMBER', 'QA_NUMBER', 'RDAY_BEGIN', 'RDAY_END', 'REQUESTED_EXPOSURE', 'ROLL', 'ROR', 'SITE', 'SUBJ_CAT', 'TITLE', 'SEARCH_OFFSET_']
-
-
-Additional query parameters
----------------------------
-
-By default, the :meth:`~astroquery.heasarc.HeasarcClass.query_object` method
-returns all entries within approximately one degree of the specified object.
-This can be modified by supplying the ``radius`` parameter. This parameter
-takes a distance to look for objects. The following modifies the search radius
-to 120 arcmin:
+    >>> from astroquery.heasarc import Heasac
+    >>> tab = Heasac.tables(keywords='chandra')
+    >>> # list the first 10
+    >>> tab[:10].pprint()
+       name                              description                           
+    ---------- ----------------------------------------------------------------
+    acceptcat Archive of Chandra Cluster Entropy Profile Tables (ACCEPT) Catal
+        aegisx  AEGIS-X Chandra Extended Groth Strip X-Ray Point Source Catalog
+    aegisxdcxo           AEGIS-X Deep Survey Chandra X-Ray Point Source Catalog
+    aknepdfcxo  Akari North Ecliptic Pole Deep Field Chandra X-Ray Point Source
+    arcquincxo Arches and Quintuplet Clusters Chandra X-Ray Point Source Catalo
+    atcdfsss82  Australia Telescope Chandra Deep Field-South and SDSS Stripe 82
+    bmwchancat                 Brera Multi-scale Wavelet Chandra Source Catalog
+    candelscxo                   CANDELS H-Band Selected Chandra Source Catalog
+    cargm31cxo          Carina Nebula Gum 31 Chandra X-Ray Point Source Catalog
+    carinaclas                 Carina Nebula Chandra X-Ray Point Source Classes
+
+If you are interested only finding the master tables, you can also set ``master`` to ``True``.
 
 .. doctest-remote-data::
 
-    >>> from astroquery.heasarc import Heasarc
-    >>> heasarc = Heasarc()
-    >>> table = heasarc.query_object(object_name, mission='rosmaster', radius='120 arcmin')
+    >>> from astroquery.heasarc import Heasac
```

Comment:
typo
```suggestion
    >>> from astroquery.heasarc import Heasarc
```

Reply:
fixed here and in other places in the rst document.